tests/conftest.py

```diff
@@ -51,3 +51,54 @@ def etl_mapping_validation_mapping_collector():
 @pytest.fixture(scope="session")
 def etl_mapping_validation_mapping_collector_unknown_prop():
     return "tests/data/etlMapping_collector_unknown_prop.yaml"
+
+
+@pytest.fixture(scope="function")
+def gitops_json():
+    with open("tests/data/gitops_test.json", "r") as f:
+        data = json.loads(f.read())
+    return data
+
+
+@pytest.fixture(scope="function")
+def gitops_json_syntax_error():
+    with open("tests/data/gitops_syntax_error.json", "r") as f:
+        data = json.loads(f.read())
+    return data
+
+
+@pytest.fixture(scope="function")
+def gitops_json_dict_error():
```

Comment:
this function and file are never used, but i think `gitops_json["graphql"]["boardCounts"][0]["graphql"] = "_notanode_count"` in `test_validate_against_dictionary` is easier to read without a fixture anyway